frontend/src/ui/player/Paella.tsx

```diff
@@ -384,6 +397,16 @@ const PAELLA_CONFIG = {
             order: 9,
             tabIndex: 9,
         },
+
+        // Data plugin
+        "es.upv.paella.cookieDataPlugin": {
+            "enabled": true,
+            "order": 0,
+            "context": [
+                "default",
+                "trimming",
+            ],
+        },
```

Comment:
I think I'd prefer the localStorage plugin: https://github.com/polimediaupv/paella-core/blob/54a45f0fb40ef7299a4bddeda086abd342d84a0d/doc/paella_core_plugins.md#esupvpaellalocalstoragedataplugin

I think local storage is the correct tool for the job, better than cookies. And yes, the preferences and data plugin docs do not mention that at all. I already thought I imagined that existing until I found the docs linked above.